.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: "tagged-release"
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  tagged-release:
+    name: "Tagged Release"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v2
```

Comment:
@tohojo do we need to set `submodules`?

```yaml
    - uses: actions/checkout@v2
      with:
        submodules: recursive
```

Reply:
Ah, neat, yeah we should add this. Didn't know there was an option to do this, and the configure script has been doing the submodule checkout, hiding the issue for the selftest. Will add, thanks for the pointer!